### PR TITLE
Upgrade storybook

### PR DESCRIPTION
**Pull request by pjnovas**
Hey I have done a some fixes to make storybook work again (it has so many stories that it was sad to see it broken). I haven't changed anything in the lib/ production side tho so it should be an smooth merge.  

Hope it helps!

### CODE REVIEW

**Comment by leandrowd** [comment]
Hi @pjnovas, thanks for updating it but what was wrong with the storybook?

**Comment by pjnovas** [comment]
well I've just discovered this lib (thanks!!) then tried http://react-responsive-carousel.js.org/storybook/ and wasn't working (which now it's fine - dns thing maybe), then I downloaded into my local and it wasn't working either (maybe not having all devDependencies into the package?, not sure).  

After some try and error I ended up upgrading the entire storybook and I thought why not send a PR of it :man_shrugging: 

Also, [codepen.io](https://codepen.io/leandrowd/pen/xLvOPd) it's not working because of SSL, right?

**Comment by leandrowd** [comment]
Cool, yeah, thanks for taking the time to update/fix it.

About the codepen, looks like https://pkgzip.com/ stopped working and the dependencies were served from there so I will need to look for a replacement. Not sure when I will be able to do it, so any help is also welcome.

File: stories/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
-import { storiesOf, action } from '@kadira/storybook';
+import {storiesOf} from '@storybook/react'
```
Comment (leandrowd) [inline stories/index.js]: Formatting (spaces) are inconsistent with the rest of the file...

**Comment by leandrowd** [comment]
I created an alternative for the codepen using webpackbin: https://www.webpackbin.com/bins/-Kxr6IEf5zXSQvGCgKBR

README.md is updated already.

**Comment by pjnovas** [comment]
hey, so I wasn't that crazy tho ... it's not working again (not sure if is just me)  

image

**Comment by leandrowd** [comment]
Hmm, please see if adding index.html to the end of the url solves the problem. I think it's a problem with the redirect.

**Comment by leandrowd** [comment]
Hey @pjnovas, thanks for the changes. 

Actually, I've found the problem with **storybook**. For some unknown reason, my `gh-pages` branch was reverted to a very old version. I have no idea why it happened but once I published it again storybooks were back to normal (http://react-responsive-carousel.js.org/storybook/).

About your PR... 
I was checking build times and your changes are increasing it by more than 80%, I don't think it's worth adding all this extra build time for an updated storybook. If you are keen on investigating why it had increased build times so much, feel free to push more changes. Otherwise, I'm going to discard this PR. 

You can check build times here: https://travis-ci.org/leandrowd/react-responsive-carousel/pull_requests - You will find that the normal time is around ~1 - 1:10 minutes. Your branch is taking from ~1:45 to 2:30 to build.

**Comment by pjnovas** [comment]
well it is installing more things; there were some assumptions for development (devDependencies) that I added in like `webpack`, `sass-loader` and `node-sass` those are pretty heavy to build.

**Comment by leandrowd** [comment]
I understand but at the current state they don't add any value, just extra
build time.

Storybook brings webpack as dependency and the style loader is working fine
in the current version.

If we decide to use something that requires bringing in node-sass then I
would agree with that but right now we don't have a way to justify the
extra time.

Thanks for taking the time to contribute. I appreciate it.

On 9 Nov. 2017 12:45 am, "Pablo Novas" <notifications@github.com> wrote:

well it is installing more things; there were some assumptions for
development (devDependencies) that I added in like webpack, sass-loader and
node-sass those are pretty heavy to build.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/leandrowd/react-responsive-carousel/pull/190#issuecomment-342821567>,
or mute the thread

**Comment by pjnovas** [comment]
sure, no problem ¯\_(ツ)_/¯

**Pull request closed without merging**